tools/shared.py

```diff
@@ -3040,15 +3040,10 @@ def delebify(buf, offset):
 
   @staticmethod
   def get_js_data(js_file, shared=False):
-    js = open(js_file).read()
```

Comment:
This function no longer holds true to name, and also doesn't use its first arg (or really its second arg, since the only caller who doesn't set `shared=True` ignores mem_align in the return anyway.

Reply:
I think this has been removed? I don't see it in the code anymore.